test/core/memory_grow.wast

```diff
@@ -0,0 +1,30 @@
+;;test that newly allocated memory (program start and memory.grow) is zeroed
+
+(module
+  (memory 0)
+  (func (export "grow") (param i32) (result i32)
+    (memory.grow (get_local 0))
+  )
+  (func $load (param $v i32) (result i32)
+    (i32.load (get_local $v))
+  )
+  (func (export "new_allocate_memory_zero") (result i32)
+    (local i32 i32 i32)
+    (set_local 0 (i32.const 0))
+    (set_local 1 (i32.const 0xfffc))
```

Comment:
Nit: for more clarity, maybe use 0x1_0000 here and compare with ge below.

Reply:
When memory is 1, I can’t read value at “0x10000”,”0xffff”, “0xfffe”,”0xfffd” , this is a known issue?